### PR TITLE
2024.06: Add smart units

### PR DESCRIPTION
**Pull request by ben-allen**

### CODE REVIEW

File: 2024/06.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     | timebox | topic | presenter |
     |:-------:|-------|-----------|
     | 60m     | Source maps progress update | Jonathan Kuperman and Agata Belkius |
+    | 60m     | Smart units progress update | Ben Allen |
```
Comment (Jack-Works) [inline 2024/06.md:123]: what is this 👀

Comment (ljharb) [reply 2024/06.md:123]: I have the same question - is there a URL for this? I'm not familiar with "smart units".

Comment (ctcpip) [reply 2024/06.md:124]: ping @ben-allen

Comment (ctcpip) [reply 2024/06.md:124]: probably this (?)  https://github.com/tc39/proposal-smart-unit-preferences

Comment (michaelficarra) [reply 2024/06.md:124]: here's the notes from last time to refresh your memory: https://github.com/tc39/notes/blob/840c700dc7fa7b9f6d0a3c208bd66b333e304717/meetings/2020-06/june-4.md#smart-unit-preferences-in-intlnumberformat-for-stage-1

Comment (ljharb) [reply 2024/06.md:124]: ping @younies; is Ben taking over this proposal?

Comment (ben-allen) [reply 2024/06.md:124]: Yes, I'm taking this one over. Apologies for the mysterious agenda addition &mdash; supporting materials TK shortly

**Review by ljharb — APPROVED** [review]
added the repo URL

**Pull request merged**